test/gst-vaapi/transcode/transcoder.py

```diff
@@ -11,45 +11,199 @@
 @slash.requires(have_gst)
 @slash.requires(*have_gst_element("vaapi"))
 @slash.requires(*have_gst_element("checksumsink2"))
+@platform_tags(ALL_PLATFORMS)
 class TranscoderTest(slash.Test):
+  requirements = dict(
+    decode = {
+      "avc" : dict(
+        sw = (ALL_PLATFORMS, have_gst_element("avdec_h264"), ("avdec_h264", "h264parse")),
```

Comment:
For the third parameter, just combine the parser and decoder into one value.  That is, `"h264parse ! avdec_h264"` instead of `("avdec_h264", "h264parse")`.  Do the same for all others including the encoders.

Reply:
modified as for comment